### PR TITLE
1446: removed BaseImage and VectorImage classes. Added tests for Image

### PR DESCRIPTION
**Pull request by romaha08**
#1446

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ImageTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4667881&bid=15623933)

**Comment by mf-code** [comment]
[4046] - https://transfer.sh/QaeL/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4046] - https://transfer.sh/aE5KP/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4046] - https://transfer.sh/2s9ct/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4046] - Allure report on Netlify: https://5df28ece118371a304e66b28--musing-swartz-6fc060.netlify.com

**Comment by OBraunEPAM** [comment]
Review is in progress.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ImageTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4667881&bid=15623933)

**Comment by mf-code** [comment]
[4221] - https://transfer.sh/h0dHO/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4221] - https://transfer.sh/gD8hs/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4221] - https://transfer.sh/B59tc/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4221] - Allure report on Netlify: https://5dfb41efb8ab4a6eced70961--musing-swartz-6fc060.netlify.com

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/CardImageData.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
     private final String altAttr;
     private final String imageClass;
     private final String text;
-    private final int width;
+    private final String width;
     private final int height;
```
Comment (Inov92) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/test/data/CardImageData.java]: height type inconsistent with width

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ImageAssert.java [hunk 2]
```diff
@@ -35,10 +35,11 @@ public ImageAssert height(Matcher<Integer> condition) {
     }
     public ImageAssert height(int height) { return height(is(height)); }
```
Comment (Inov92) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/asserts/ImageAssert.java]: Inconsistent logic of chechking height regarding width

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Image.java [hunk 3]
```diff
@@ -21,9 +23,21 @@
     @JDIAction(value = "Get '{name}' image height", level = DEBUG)
     public int height() { return getInt("height", uiElement); }
     @JDIAction(value = "Get '{name}' image width", level = DEBUG)
-    public int width() { return getInt("width", uiElement); }
+    public String width() { return uiElement.attr("width"); }
     @JDIAction(value = "Get '{name}' image alt", level = DEBUG)
     public String alt() { return uiElement.attr("alt"); }
+    @JDIAction(value = "Get '{name}' vector image tag attribute by its name", level = DEBUG)
```
Comment (Inov92) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Image.java]: Vector image is obsolete. Logging message update required.

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ImageTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4667881&bid=15623933)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/ImageTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4667881&bid=15623933)

**Comment by mf-code** [comment]
[4296] - https://transfer.sh/I8PIX/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4296] - https://transfer.sh/biymq/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4296] - https://transfer.sh/FgayQ/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4296] - Allure report on Netlify: https://5dfccd9de91cba36f6f913b4--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4298] - https://transfer.sh/8NUWN/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4298] - https://transfer.sh/tWrrM/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4298] - https://transfer.sh/Dw6ER/jdi-light-examples.tar.gz

**Pull request merged**